prisma.config.ts

```
@@ -0,0 +1,16 @@
+// npm install --save-dev prisma dotenv
+import "dotenv/config";
+import { defineConfig, env } from "prisma/config";
+
+export default defineConfig({
+  schema: "prisma/schema.prisma",
+  migrations: {
+    path: "prisma/migrations",
+  },
+  engine: "classic",
+  datasource: {
+    url: env("DATABASE_URL"),
```

ESLint

[error] Replace `"DATABASE_URL"` with `'DATABASE_URL'`

(prettier/prettier)